src/frontend/components/Chat/SharedChatComponents/InputBar/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { Box, Button, Text, TextInput } from 'grommet';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { SendButtonSVG } from 'components/SVGIcons/SendButtonSVG';
+
+const ButtonStyled = styled(Button)`
+  padding: 0;
+`;
+
+interface InputBarProps {
+  handleUserInput: (inputText: string) => boolean;
+  isChatInput: boolean;
+  placeholderText: string;
+}
+
+export const InputBar = ({
```

Comment:
it seems you condition a lot of things with `isChatInput` maybe declare variable before using these in jsx

Reply:
IMO defining directly in the props with ternary is more readable